src/components/images/imageLoader.js

```diff
@@ -12,13 +12,6 @@ import 'css!./style';
         fillImageElement(elem, source);
     }
 
-    export function getImageBlurhashStr(hashes, tags) {
```

Comment:
@JustAMan You shouldn't have removed this one, because we still need to implement this check in listview

Reply:
Wait, you cannot use same one-liner I used in other place?.. it literally checks if `hashes` map is not null and gets a value from it... `(hashes || {})[tags]` does _exactly the same_ and is shorter than this function signature 😛